### PR TITLE
Add VitaminA international unit

### PR DESCRIPTION
**Pull request by neutmute**
Hi Andreas - As discussed in #126, this PR adds VitaminA International Unit type.

I didn't add other Vitamin's or conversions to micrograms yet - keeping things simple to start off with.

Let me know how this looks!

### CODE REVIEW

**Comment by neutmute** [comment]
One thought - are these properties correct in the json:

            "SingularName": "InternationalUnit",
            "PluralName": "InternationalUnits",

Or should the words be separated by spaces?

            "SingularName": "International Unit",
            "PluralName": "International Units",

**Comment by anjdreas** [comment]
Looks good to me!
I'll await merging in assuming you want to add more units to VitaminA. You decide if you want to do VitaminD in a separate PR or in the same one, does not matter to me just let me know when things are ready to merge in and I'll review it.

Singular/PluralName cannot have space, as the string is used to generate C# enum type and C# method/property names.

**Comment by neutmute** [comment]
Excellent. 

re: more additions, sorry - I gave the wrong impression in my comment that I would submit further commits for conversions/Vitamin D but this should suffice for my projected needs and the PR is ready to merge if satisfactory.

TLDR:
* Vitamin D isn't part of the nutrition info I need to store but more an example to show that IU's from different molecules cannot be mingled.
* I won't need the microgram conversion either as far as I know. Using the IU type and being able to divide/multiply/add is sufficient for my project needs.

**Comment by anjdreas** [comment]
Nuget 3.24 out.

**Comment by neutmute** [comment]
awesome - thanks
